web-client/src/views/Public/TrialSessions/PublicTrialSessionDetails.tsx

```diff
@@ -158,15 +158,7 @@ function NonMobileOpenCases({
                   <CaseIcons formattedCase={publicCase} />
                 </td>
                 <td>
-                  <span
-                    className={classNames({
-                      'margin-left-2':
-                        publicCase.inConsolidatedGroup &&
-                        !publicCase.isLeadCase,
-                    })}
-                  >
-                    <CaseLink formattedCase={publicCase} />
-                  </span>
```

Comment:
I'm not sure why I had this indentation. It isn't in the design.

Reply:
Some tables have it, some don't. Usually depends on whether the consolidated cases are grouped together or not